kotlin/ktor/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM gradle:4.10-jdk8
```

Comment:
Why this version ? The latest available is not the `5` ?

Reply:
Had problem in a more complex previous project updating to gradle 5. So just out of lazyness.
Made the change here and everything went smoothly.